lib/store/peer.ex

```diff
@@ -73,11 +73,12 @@ defmodule Elixium.Store.Peer do
           {:ok, peers} ->
             peers =
               [peer | :erlang.binary_to_term(peers)]
-              |> Enum.uniq()
+              #add here
```

Comment:
Looks like something is missing here

Reply:
Was legacy from working with the own ip filtering, removed